layouts/shortcodes/install-pulumi.html

```diff
@@ -5,9 +5,9 @@
 */}}
 <div class="mdl-tabs os-tab-tabs">
     <div class="mdl-tabs__tab-dir os-tab-tabstrip">
-        <a data-track="oschoose-macOs" class="mdl-tabs__tab os-tab is-active">macOS</a>
```

Comment:
These are causing a brief blink on reload, so since we seem to be setting them in all cases, it seemed fine to remove them.